score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java

```diff
@@ -99,7 +99,9 @@
 
     public static final String ENTERPRISE_MODE = "ENTERPRISE_MODE";
 
-    protected Map<String, Serializable> contextMap;
+    private static final String SC_NESTED_FOR_PARALLELISM_LEVEL = "SC_NESTED_FOR_PARALLELISM_LEVEL";
+
+    protected Map<String, Serializable> contextMap = new HashMap<>();
```

Comment:
this initialization was not merged properly during conflict resolution, it moved to the constructor and no longer exists.